lib/commands/project.js

```diff
@@ -1042,6 +1041,48 @@ project.getMainFile = function(cb) {
   }
 };
 
+//-----------------------------------------------------------------------------
+/**
+ * Description - Downloads the source code for a specific project.
+ * @param {string} projectName - name of the project to retrieve logs from.
+ * @param {string} output - filename to download the tarball at
+ * @param {function} cb Callback invoked with log data.
+ */
+// -----------------------------------------------------------------------------
+project.download = function (projectName, output, cb) {
+  projectController.find({
+      userId: userConfig.data.userId
+    },
+    function (err, projects) {
+      if (err) {
+        err = error.handleApiError(err, 'GET_PROJECT_SOURCE', cb);
+        if (err.length > 0) return cb(err);
+      }
+
+      if (projects.length === 0) {
+        modulus.io.error('You currently have no projects. You can create one with "project create".');
```

Comment:
This seems like a valid error message for `projects.length === 0`, am I missing something?